### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by enjoy-binbin**
Correct minor spelling typos

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@enjoy-binbin

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/dataclasses.py
Comment (terryjreedy) [inline Lib/dataclasses.py]: slightly and performant are correct

File: Lib/plistlib.py
Comment (terryjreedy) [inline Lib/plistlib.py]: declarations

**Comment by terryjreedy** [comment]
Why did you close this?  The CLA?  For spelling corrections, we merely prefer the CLA, as the change is from a 'creative' misspelling to the dictionary standard.

File: Lib/idlelib/config.py
Comment (terryjreedy) [inline Lib/idlelib/config.py:19]: 'useable' is an accepted alternative spelling of 'usable', in USA as well as UK.  I am leaving the change.  Maybe the doc group will someday rule on such issues.

File: Lib/wsgiref/validate.py
Comment (terryjreedy) [inline Lib/wsgiref/validate.py]: 'compliancy' is an alternate spelling of 'compliance' which may or may not have a narrower or slightly different meaning, depending on the source, and which may be preferred in certain contexts or narrow communities.  Since it is possible that the author of the sentence consciously chose it, I am reverting.
DONE

**Comment by enjoy-binbin** [comment]
@terryjreedy Hi. Thanks for your reply. I am new. That means a lot to me.
The only reason i closed it because the CLA... I have applied for the sign, juse wait the result
So that i can reopen again. Avoid disturbing you guys. But it seems that you have been mentioned

Also thanks again for your update change. Feel free to change the error one.
About the useable, since it is right one, i am going to revert it, no need to make more changes

**Review by terryjreedy — APPROVED** [review]
Closing blocks CI.  We should ask people to say when have signed, as it takes a full business day, sometimes more, to get recorded.

After I made a few changes, I will merge when CI passes.  Do not make any more changes.

**Comment by terryjreedy** [comment]
Test failure:
```
Generated files not up to date. Perhaps you forgot to run make regen-all ;)
 M Python/importlib_external.h
```
When the test machines run configure, the changed configure.ac results in generated files looking out of date, or something like that.  Hence 'make regen-all' needs to be run and any changes committed. 

If this can be done on Windows with git bash, I don't know how, and it would have to be done for each backport.  So configure.ac changes are best done separately.  I will revert the definition correction.

**Comment by terryjreedy** [comment]
Reverting configure.ac did not work, so I made sure updated to latest main.

**Comment by cjw296** [comment]
@terryjreedy - be careful with this one, I spotted some changes that weren't good.

**Comment by terryjreedy** [comment]
@cjw296  Please be specific.  A couple of hours ago I reviewed carefully and already reverted, corrected, or checked everything I though questionable.

**Comment by terryjreedy** [comment]
@cjw296 I recheck and found this: in ssl.py, 'woth' should by 'with', not 'worth'.  Fixed.  I think others looks good in context.

**Comment by terryjreedy** [comment]
Bin-bin, If you (or anyone else) are on Linux and can run 'make regen-all' and commet whatever that changes, please try that.

**Comment by ericvsmith** [comment]
I'm not on Linux right now, so I can't help with 'make regen-all', sorry.

But all of the corrections look good to me. Thanks, @enjoy-binbin and @terryjreedy.

**Review by ericvsmith — APPROVED** [review]
I'm not on Linux right now, so I can't help with 'make regen-all', sorry.

But all of the corrections look good to me. Thanks, @enjoy-binbin and @terryjreedy.

**Comment by enjoy-binbin** [comment]
@terryjreedy It seems that `re-gen` was ok. I was trying to solve that one half hour ago.
But i only have a centos and i encounter some problems.
Now look like the hardest part has been solved.

Sorry for that, i didn't expect to cause this problem... and others minor fix
Thanks a lot, it must have cost you a lot of time

**Comment by terryjreedy** [comment]
On corementorship, Guido said how to rebuild on windows: `PCbuild\build -r`.  `importlib_external.h` was rebuilt, followed by errors that disappeared on rerun.  I will redo the the misspellings I reverted and is that does not create a problem, merge.

**Comment by terryjreedy** [comment]
Yes, more time than expected.  At least I learned that rebuilding can be done on Windows.  I have seen the make message  before and assumed I had to have a make system.  Another core def suggested that the error message should be more informative.

I expected that auto backport would fail even though most of these misspelling either originated with or were backported at least to 3.10.  If you do a 3.10 backport that passes tests , I will review and merge.  Using the cherry_picker module is easier than git cherry-picker as it automates more.  Whether and how are up to you.

**Pull request merged**
